archai/nlp/nvidia_transformer_xl/mem_transformer.py

```diff
@@ -732,16 +741,15 @@ def _create_params(self):
         # default attention
         if self.attn_type == 0:
             self.pos_emb = PositionalEmbedding(self.d_model)
-            self.r_w_bias = nn.Parameter(torch.Tensor(self.n_head, self.d_head).zero_())
-            self.r_r_bias = nn.Parameter(torch.Tensor(self.n_head, self.d_head).zero_())
+            for i, _ in enumerate(self.layers):
```

Comment:
Let's check memory footprint/param count

Reply:
New:
```
{'n_all_param': 191965658, 'process_mem': 2927493120, 'test_ppl': 22.33852467331828, 'test_ppl_nomem': 30.888911615569228, 'train_elapsed': 2199.6180336356165}
```

Old:
```
{'n_all_param': 191950298, 'process_mem': 2928812032, 'test_ppl': 22.509031654014887, 'test_ppl_nomem': 31.102925247305357, 'train_elapsed': 2212.331364182631}
```

Reply:
Memory and training time seems to be pretty much the same, and we were able to improve the baseline by 0.75%.